app/components/MeToken/createMetoken.tsx

```
@@ -158,14 +133,42 @@ export default function CreateMetoken() {
+
       try {
         const transaction = prepareContractCall({
           contract: metokenFactoryBase,
           method: 'create',
           params: [formValues.name, formValues.symbol, formValues.address],
         });
-        sendTransaction(transaction);
-        setIsLoading(false);
+
+        sendTransaction(transaction, {
+          onSuccess: (result: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)